model/bin/comp.ite

```diff
@@ -0,0 +1,213 @@
+#!/bin/bash
```

Comment:
All other comp/link platform specific files have been removed from the WW3 develop branch.  Should these be included in this PR or removed?

Reply:
Hi  @JessicaMeixner-NOAA this will be remove.